examples/inverse/plot_compute_mne_inverse_epochs_in_label.py

```diff
@@ -64,11 +64,6 @@
 stcs = apply_inverse_epochs(epochs, inverse_operator, lambda2, method, label,
                             pick_ori="normal", nave=evoked.nave)
 
-stc_evoked = apply_inverse(evoked, inverse_operator, lambda2, method,
-                           pick_ori="normal")
-
-stc_evoked_label = stc_evoked.in_label(label)
```

Comment:
Is the in_label present elsewhere in the doc ?

Reply:
11 lines below this